playbook/app/pb_kits/playbook/pb_title/title.rb

```diff
@@ -3,19 +3,20 @@
 module Playbook
   module PbTitle
     class Title < Playbook::KitBase
+      prop :color, type: Playbook::Props::Enum,
+                   values: %w[default light lighter success error link],
+                   default: "default"
```

Comment:
Here too: Color default should be set to nil since default is one of the color values that can be passed to the kit.